presto-product-tests/bin/run_on_docker.sh

```diff
@@ -159,6 +168,10 @@ docker version
 
 stop_all_containers
 
+if [[ "$CONTINUOUS_INTEGRATION" == 'true' ]]; then
```

Comment:
This is ok, but seems a bit Travis-specific. Can you try checking if stdout is a tty?
```shell
test -t 1 || prefetch_images_silently